snippets/installing-subnet-requirements.adoc

```diff
@@ -0,0 +1,27 @@
+// Text snippet included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing-openstack/installing-openstack-installer-custom.adoc
+// * installing/installing-openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing-openstack/installing-openstack-installer-restricted.adoc
+// * installing/installing-rhv/installing-rhv-default.adoc
+// * installing/installing-rhv/installing-rhv-customizations.adoc
+
+
+//include::snippets/installing-subnet-requirements.adoc[]
+
+:_content-type: SNIPPET
+
+* If you are using L2 subnets, you ensured that either your clusters will run on dedicated subnets or that every cluster on a given subnet has a unique Virtual Router Identifier (VRID). Running clusters with the same VRID causes broadcast storms and connection reset errors.
++
+The installer process assigns VRIDs based on cluster IDs. Even if you change the cluster ID after installation, you cannot change the VRID assigned to it.
++
+To check the VRIDs, run the following command for all clusters in a given subnet, including any clusters you intend to install:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run quay.io/openshift/origin-baremetal-runtimecfg:{product-version} vr-ids <cluster_ID>
```

Comment:
It doesn't seem very frequent to see prerequisites with commands (understandable... ), but this seemed the best place to add it.  

This command comes from here "https://access.redhat.com/solutions/6450581"  though, so another possibility would be adding a link there. But the reader will need to be logged to see it and click, so not as convenient. 

Thoughts?

Reply:
Also I expect "{product-version} resolves to 4.7, 4.8, etc.  At the moment it resolves to "Branch Build" but I think that's fine?